polkadot-parachains/src/service.rs

```diff
@@ -28,7 +27,9 @@ use cumulus_primitives_core::{
 	relay_chain::v1::{Hash as PHash, PersistedValidationData},
 	ParaId,
 };
+pub use polkadot_primitives::v1::{AccountId, Balance, Block, Hash, Nonce};
```

Comment:
This is not correct! 

We want to use the parachain specific types and not the types from Polkadot.

Reply:
Makes sense! We probably want to factor them out as a module like to make them compatible. for now defined them as separate types.